Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.22.2
+WORKDIR /app
+COPY go.mod go.sum ./
+RUN go mod download
+COPY *.go ./
+COPY server ./server
+COPY client ./client
+COPY assets ./assets
+COPY config ./config
+COPY .env ./
```

Comment:
Environment variables should not be in the image but rather set by that which runs the image